ci/test_python.sh

```diff
@@ -54,5 +55,21 @@ pytest \
  --cov-report=term \
  tests
 
+rapids-logger "pytest cuvs-bench"
+pushd python/cuvs_bench/cuvs_bench
+pytest \
+ --cache-clear \
+ --junitxml="${RAPIDS_TESTS_DIR}/junit-cuvs.xml" \
+ --cov-config=../.coveragerc \
+ --cov=cuvs \
+ --cov-report=xml:"${RAPIDS_COVERAGE_DIR}/cuvs-bench-coverage.xml" \
+ --cov-report=term \
+ tests
+
+rapids-logger "cuvs-bench e2e test with synthetic data"
+python -m cuvs_bench.get_dataset --dataset test-data --dataset-path datasets
```

Comment:
Just found out that `click` has a `Clirunner` for testing the CLI in pytest in a more robust manner, so changing these calls to that